### PR TITLE
ref: Remove python normalizer

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: src/sentry/constants.py [hunk 1]
```diff
@@ -123,9 +123,10 @@ def get_all_languages():
 DEFAULT_SORT_OPTION = 'date'
 
 # Setup languages for only available locales
-LANGUAGE_MAP = dict(settings.LANGUAGES)
```
Comment (mitsuhiko) [inline src/sentry/constants.py]: Why is this gone?

**Review by mitsuhiko — COMMENTED** [review]
Looks good other than the constant that was removed

File: src/sentry/constants.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/constants.py]: I was looking through unused constants and removed things that I didn't see used in sentry, getsentry and sentry-plugins

**Comment by mitsuhiko** [comment]
Will review after the tests are passing. There is some stuff not entirely right right now.

**Comment by untitaker** [comment]
@mitsuhiko the battle with packaging infra is over and the tests pass now.

**Comment by mitsuhiko** [comment]
@untitaker now you just need to fix the conflicts! :D

**Review by mitsuhiko — APPROVED** [review]

**Pull request merged**